src/data/location.json

```diff
@@ -261,6 +281,24 @@
         },
         "scale": 300
       }
+    ],
+    "infoMarkers": [
+      {
+        "markerId": 6,
+        "position": {
```

Comment:
This marker is misplaced. The location is looking over the sunken garden and it's a infoMarker but has been placed on a wall instead of down in the sunken garden.